rules/namespace.go

```diff
@@ -119,3 +122,49 @@ func (nss Namespaces) Version() int { return nss.version }
 
 // Namespaces returns the list of namespaces.
 func (nss Namespaces) Namespaces() []Namespace { return nss.namespaces }
+
+// Schema returns the given Namespace in protobuf form
+func (s NamespaceSnapshot) Schema() *schema.NamespaceSnapshot {
```

Comment:
Can we move the methods closer to where the other methods are defined for this type? Makes it easier to read/find stuff.